docs/navigate/migration-guide/toc.yml

```diff
@@ -26,7 +26,7 @@ items:
           - name: Migration sample
             href: ../../azure/migration/appmod/sample.md?toc=/dotnet/navigate/migration-guide/toc.json&bc=/dotnet/breadcrumb/toc.json
           - name: Troubleshooting
-            href: ../../azure/migration/appmod/faq.md?toc=/dotnet/navigate/migration-guide/toc.json&bc=/dotnet/breadcrumb/toc.json
+            href: /dotnet/core/porting/github-copilot-app-modernization-faq?toc=/dotnet/azure/toc.json&bc=/dotnet/breadcrumb/toc.json?toc=/dotnet/navigate/migration-guide/toc.json&bc=/dotnet/breadcrumb/toc.json
```

Comment:
@BillWagner For next time, let's convert these to relative links before merging. I'll fix.